certs/test/cert-over-max-altnames.pem

```diff
@@ -0,0 +1,63 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
These new files need to be added to the `certs/test/include.am` otherwise they will not be in the `make dist`.

Reply:
I have included it in the latest commit.